### DISCUSSION TITLE
Discharge.nc won't open in Panoply

### DISCUSSION THREAD

**Discussion by emresalkim** [mhm-ufz/mHM#46] [Q&A] [answered]
Hello mHM Dev. Team,

Even though I made a fresh install of Ubuntu (WSL2) and mHM and Panoply too (Windows), whenever I want to visualize the Discharge.nc at Panoply I recieve this error: There was an error opening the dataset: DataObject doesn't start with OHDR. Tried the Google way but couldn't fix it. 

What should I do?

**Comment by pallavhydro** [comment]
Hi @emresalkim 

I am sorry to hear you are having trouble visualising the mhm output.

Apart from Panoply, one more platform to visualise netCDF would be [ncview](https://anaconda.org/conda-forge/ncview). Since you worked out the WSL2 setup, I am assuming you have conda environment for mhm. I think it should be fine to activate that environment and then install ncview using 

```
conda install ncview 
```

Once the installation works, try opening netcdf file using 

```
ncview discharge.nc &
```

Let us know if this works for you.

Cheers
Pallav

**Comment by emresalkim** [comment, marked as answer]
Dear @pallavhydro 

Thank you for your answer. Unfortunately ncview also won't work.

It says: _Note: could not open file /home/emre/.ncviewrc for reading
netcdf_varindex_to_name: error on nc_inq_var call_

Regards,
Emre

**Reply by StephanThober** [reply]
Dear @emresalkim ,

I have never experienced any issue with the discharge.nc file. This error from ncview suggests that there is some variable it wants to read that may not exist. Could you upload your discharge.nc file here?

Best,
Stephan

**Reply by emresalkim** [reply]
Dear @StephanThober ,

I fixed the issue after making a clean install of mHM without used Miniconda. I don't know what was the problem there back then, however it is working without any issues now. Thank you for your reply!

Yours,
Emre

**Comment by pallavhydro** [comment]
Hi @emresalkim 

Have you checked whether you have a .ncviewrc at the location then? 


A. If you don't have one, create a simple text file - copy the following content to it - and save it as .ncviewrc.


```
-1 "STRINGLIST_SAVE_FILE_VERSION" INT 1
0 "NCVIEW_STATE_FILE_VERSION" INT 1
1 "CMAP_3gauss" INT 1
2 "CMAP_detail" INT 1
3 "CMAP_ssec" INT 1
4 "CMAP_bright" INT 1
5 "CMAP_banded" INT 1
6 "CMAP_rainbow" INT 1
7 "CMAP_jaisnb" INT 1
8 "CMAP_jaisnc" INT 1
9 "CMAP_jaisnd" INT 1
10 "CMAP_blu_red" INT 1
11 "CMAP_manga" INT 1
12 "CMAP_jet" INT 1
13 "CMAP_wheel" INT 1
14 "CMAP_3saw" INT 1
15 "CMAP_bw" INT 1
16 "CMAP_default" INT 1
17 "CMAP_extrema" INT 1
18 "CMAP_helix" INT 1
19 "CMAP_helix2" INT 1
20 "CMAP_hotres" INT 1
```

B. If you have one, then make sure you have reading access to it.

Note: files like .bashrc, .ncviewrc, etc. are so called **run command (rc)** files which are set of commands run every time you open  the corresponding program (bash, ncview, etc). As you may already know, the files that start with a . are hidden files.

Keep us posted
Pallav

**Comment by emresalkim** [comment]
Dear Pallav,

Now the error turned into this: Note: could not open file /home/emre/.ncviewrc for reading
Error: Can't open display:

Can this all be because of Windows 11?
Should make a clean installation of Windows 10?

**Comment by pallavhydro** [comment]
Hi @emresalkim 

You are calling a display from linux command line, which requires [X11 display protocol.](https://kb.thayer.dartmouth.edu/article/336-x11-for-windows-and-mac) 

I use XQuartz for my Mac. Apparently Xming is its counterpart for Windows. After Xming is installed, you might have (if needed) to make some changes in the setting of Xming and hopefully that enables ncview to open the display for you.

Pallav

**Reply by emresalkim** [reply]
Hello @pallavhydro Thank you once more. I was able to ncview .nc files at this pc a week ago or so. Win 11 already has a Terminal in it. However, now it just doesn't work neither does Panoply. Gonna try switching to Win 10 and make another clean install. Let's see if this works. I believe it is Win 11 that creates the problem. My friends running Win 10 are not having these problems.

**Comment by pallavhydro** [comment]
Hi @emresalkim .

Just wanted to update that ...

This issue has been solved in the latest release of mHM.
Re-install of mHM should solve it, ie. if you have conda then `conda install mhm`
Let us know whether the discharge.nc opens with Panoply.
Thanks.

Pallav

**Comment by neginsa1994** [comment]
hi ... i have error opening data when i want open GRIB data in panpoly ...."There was an error opening the dataset"  ...what should i do?

**Reply by rohinikumar82** [reply]
We don't use GRIP data in mHM. You can convert your data from GRIP to netcdf (I guess, there will be couple of programs to do) and then visualise it in Panpoly
